webpack.config.js

```diff
@@ -90,6 +89,71 @@ module.exports = ({production, server, extractCss, coverage} = {}) => ({
   },
   plugins: [
     new AureliaPlugin(),
+    new ModuleDependenciesPlugin({
```

Comment:
Well, I'd like to try without. 😃 
Since somehow this is the point of it all.. We can comment it and leave it in, of course.